lib/specs/latest.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash');
+const previousSpec = require('./v1');
+
+const previousKnownHelpers = previousSpec.knownHelpers;
+const previousTemplates = previousSpec.templates;
+const previousRules = previousSpec.rules;
+
+let knownHelpers;
+let templates;
+let rules;
+
+knownHelpers = [];
+
+knownHelpers = _.union(previousKnownHelpers, knownHelpers);
```

Comment:
Hmm if we want to remove helpers in the latest version, how does that work with `.union`?

Are there any tests who reflect how the merge of the rules works?

Reply:
No tests yet for this case.  But you're right, currently with wouldn't work. It probably needs more logic after the `_.union` to mark helpers as `deleted` or so in a newer version and remove them from the array. Or something like this... Will have a 🤔about this.